qemu/tests/cfg/boot_nbdimage_with_qsd.cfg

```diff
@@ -0,0 +1,41 @@
+- boot_nbdimage_with_qsd:
+    virt_test_type = qemu
+    type = boot_nbdimage_with_qsd
+    required_qemu = [6.0, )
+    start_vm = no
+    kill_vm = yes
+    # prepare an installed os image via converting
+    convert_source = ${images}
+    convert_target = stg1
+    image_name_stg1 = "images/stg1"
+    image_format_stg1 = raw
+    enable_nbd_stg1 = no
+    storage_type_stg1 = filesystem
+    remove_image_stg1 = 'yes'
+    qsd_create_image_stg1 = no
+    qsd_remove_image_stg1 = yes
+    # Declare QSD (name), multi QSD separated with blank
+    qsd_namespaces = "qsd1"
+    # Declare specified QSD contains which images
+    qsd_images_qsd1 = "stg1"
+    qsd_daemonize_qsd1 = yes
+    # Run QSD in daemon mode ,default is n
+    # Force create QSD even if find same name QSD,default is yes
+    qsd_force_create_qsd1 = yes
+    # Auto adding --pidfile into QSD command line, default is yes
+    qsd_enable_pidfile_qsd1 = yes
+    # Raw command line of QSD daemon,QSD concat them
+    qsd_image_format = {"driver":"raw","read-only":false,"discard":"unmap"}
+    # Declare one image export with nbd inet
+    qsd_image_export_stg1 = {"type":"nbd"}
+    qsd_image_export_nbd_stg1 = {"type":"unix","path":"/var/run/nbd.sock"}
+    # NBD client based on inet
+    nbd_image_tag = nbd1
+    nbd_server_stg1 = ''
+    nbd_server_nbd1 = ''
+    force_create_image_nbd1 = no
```

Comment:
Remove these 3 lines, note force_create_image takes effect only when nbd1 is defined in images

Reply:
Need 
```
nbd_server_stg1 = ''
nbd_server_nbd1 = ''
```
Or the guest will boot from the original image(The converted source NBD image).

Reply:
The unix socket is also over nbd backend. Why do we not need a nbd storage backend?

Reply:
Because we just setup an internal nbd server, everything is configurable, and there is no remote nbd server. For an internal nbd server, we can configure it in cfg instead of a introducing the nbd backend. It's easy, unless the nbd storage backend is a must. e.g.
we must convert a nbd image to a local image, and then export the local image, but refer to your case, I don't think so.